CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+## Hello Contributors!
+
+Thanks for your interest!
+We're so glad you're here.
+
+### Important Resources
+
+#### bugs: [https://github.com/tinkerbell/terraform-provider-tinkerbell/issues](https://github.com/tinkerbell/terraform-provider-tinkerbell/issues)
+
+### Code of Conduct
+
+Please read and understand the code of conduct found [here](https://github.com/tinkerbell/.github/blob/master/CODE_OF_CONDUCT.md).
+
+### DCO Sign Off
+
+Please read and understand the DCO found [here](docs/DCO.md).
+
+### Environment Details
+
+Building is handled by `make`, please see the [Makefile](Makefile) for available targets.
+
+### How to Submit Change Requests
+
+Please submit change requests and / or features via [Issues](https://github.com/tinkerbell/terraform-provider-tinkerbell/issues).
+There's no guarantee it'll be changed, but you never know until you try.
+We'll try to add comments as soon as possible, though.
+
+### How to Report a Bug
+
+Bugs are problems in code, in the functionality of an application or in its UI design; you can submit them through [Issues](https://github.com/tinkerbell/terraform-provider-tinkerbell/issues).
+
+## Code Style Guides
```

Comment:
Should something be mentioned here? At least that there is no style guides at the moment or link to something general like https://github.com/golang/go/wiki/CodeReviewComments ?

Reply:
> LGTM, just one comment. Someone feel free to merge, I don't know how those files look in other repos, so can't say for sure this is right.

I'm doing similar PRs to all the other repos, :D

Reply:
> Should something be mentioned here? At least that there is no style guides at the moment or link to something general like [golang/go/wiki/CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments) ?

Lets leave blank for this PR and then we can follow up with *some* guide lines, but also this maybe repo specific so I'd like the "common stuff" pr first.